projects/batfish/src/org/batfish/grammar/cisco/Cisco_bgp.g4

```diff
@@ -484,8 +507,7 @@ null_bgp_tail
       | FALL_OVER
       | LOCAL_V6_ADDR
       | LOG_NEIGHBOR_CHANGES
-      | MAXIMUM_PATHS
```

Comment:
Both instances of MAXIMUM_PREFIX should have been removed if you are putting maximum_prefix_bgp_tail in bgp_tail